modules/arm_plugin/demos/coco_detection_android_demo/app/src/main/java/org/intel/openvino/MainActivity.java

```diff
@@ -0,0 +1,273 @@
+            "bear", "zebra", "giraffe", "hat", "backpack", "umbrella", "shoe", "eye glasses",
+            "handbag", "tie", "suitcase", "frisbee", "skis", "snowboard", "sports ball", "kite",
+            "baseball bat", "baseball glove", "skateboard", "surfboard", "tennis racket", "bottle",
+            "plate", "wine glass", "cup", "fork", "knife", "spoon", "bowl", "banana", "apple",
+            "sandwich", "orange", "broccoli", "carrot", "hot dog", "pizza", "donut", "cake", "chair",
+            "couch", "potted plant", "bed", "mirror", "dining table", "window", "desk", "toilet",
+            "door", "tv", "laptop", "mouse", "remote", "keyboard", "cell phone", "microwave", "oven",
+            "toaster", "sink", "refrigerator", "blender", "book", "clock", "vase", "scissors",
+            "teddy bear", "hair drier", "toothbrush", "hair brush"
+    };
+
+    private void copyFiles() {
+        String[] fileNames = {MODEL_BIN, MODEL_XML, PLUGINS_XML};
+        for (String fileName: fileNames) {
+            String outputFilePath = modelDir + "/" + fileName;
+            File outputFile = new File(outputFilePath);
+            if (!outputFile.exists()) {
+                try {
+                    InputStream inputStream = getApplicationContext().getAssets().open(fileName);
+                    OutputStream outputStream = new FileOutputStream(outputFilePath);
+                    byte[] buffer = new byte[5120];
+                    int length = inputStream.read(buffer);
+                    while (length > 0) {
+                        outputStream.write(buffer, 0, length);
+                        length = inputStream.read(buffer);
+                    }
+                    outputStream.flush();
+                    outputStream.close();
+                    inputStream.close();
+                } catch (Exception e) {
+                    Log.e("CopyError", "Copying model has failed.");
+                    System.exit(1);
+                }
+            }
+        }
+    }
+
+    private void createRandomColor(int length) {
+        Random r = new Random(999);
+        randomColor = new Scalar[length];
+
+        for (int i = 0; i < length; i++) {
+            randomColor[i] = new Scalar(r.nextDouble() * 255, r.nextDouble() * 255, r.nextDouble() * 255);
+        }
+    }
+
+    private void processNetwork() {
+        // Set up camera listener
+        mOpenCvCameraView = (CameraBridgeViewBase) findViewById(R.id.CameraView);
+        mOpenCvCameraView.setVisibility(CameraBridgeViewBase.VISIBLE);
+        mOpenCvCameraView.setCvCameraViewListener(this);
+        mOpenCvCameraView.setCameraPermissionGranted();
+        mOpenCvCameraView.enableFpsMeter();
+        mOpenCvCameraView.setMaxFrameSize(320, 240);
+
+        // Initialize model
+        copyFiles();
+        Core core = new Core(modelDir + "/" + PLUGINS_XML);
+        Model net = core.read_model(modelDir + "/" + MODEL_XML);
+        System.out.println("load ok...");
```

Comment:
Println probably ignored by logcat. Take a look at `Logger`:

```java
private static final Logger logger = Logger.getLogger(MainActivity.class.getName());
//...
logger.info("load ok...");
```

Reply:
Thanks, I used the `Log` class uniformly. :)